azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -16,6 +16,7 @@
 
 package azkaban.project;
 
+import static azkaban.Constants.*;
```

Comment:
Why import * here and then import specifics?

Reply:
thanks, removed.